deploy/azure/install-agent.sh

```diff
@@ -1,6 +1,12 @@
 #!/bin/bash
 set -euxo pipefail
 
+# Delete password, stop and disable ssh
+sudo passwd --delete cloudbeat &
+sudo systemctl disable --now ssh &
+sudo systemctl mask ssh.service &
+sudo killall sshd &
```

Comment:
What if one of the commands fails? 
Could running all commands in the background concurrently potentially lead to race conditions?

Reply:
If they fail, we just have to accept we can't do much. Re-ordered them around to minimize the scenario where something fails.